### ISSUE TITLE
add batchnormalization to MNIST model

### ISSUE THREAD

**Issue by sari-rev00** [sari-rev00/pytorch_image_classifier#1]
insert batchNorm2d between conv layer and relu
